### PR TITLE
template: Add more style guides to template

### PR DESCRIPTION
**Pull request by seisman**
**Description of proposed changes**

Fixes #

**PRs for new software/tutorials**

- [x] Software metadata is complete
- [x] The "last updated date" in the metadata page is up-to-date
- [x] All links works
- [x] You're listed as the author(s) in the metadata page
- [x] Anyone who have made significant contributions (e.g., review) to
      the tutorials are listed in the meteda page

**PRs for bugs/improvements**

- [x] You're listed as the proofreader in the metadata page

### CODE REVIEW

File: source/template/style.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+文件风格
+--------
+
+1.  所有教程均采用 `reStructuredText <https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html>`__
+    语言写成
+2.  节标题、二级节标题和三级节标题，分别用 ``=``、``-`` 和 ``^`` 符号标识
+3.  所有 Bash 命令前应加上 shell 提示符 ``$`` 以表示该命令为 shell 命令
+4.  中文字符与英文字符之间应加上空格，如 ``中文 ABC 中文`` 而非 ``中文ABC中文``
+5.  图片应在保证清晰度的前提下尽可能小。可以使用 `squoosh 在线工具 <https://squoosh.app/>`__
+    进行压缩
```
Comment (seisman) [inline source/template/style.rst:33]: Anything more?

File: source/template/introduction.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
 简介
 ====
 
-本文档是 **地震“学”软件** 的教程模板。该目标规定了教程中所需包含的内容，
+本文档是\ **地震“学”软件**\ 的教程模板。该模板规定了软件教程中所需包含的内容，
```
Comment (core-man) [inline source/template/introduction.rst:4]: Why sometimes you add `\` before and after the whitespace?

File: source/template/style.rst [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+   - 其它可选教程
+
+     - 使用说明
+     - 实战练习
+
+3. 关于文件名的规则：
+
+   - 文件名一律采用小写字母
+   - 若文件名中含多个单词，应使用连字符 (hyphen) ``-`` 连接
+   - 文件名应尽量使用单词全称，避免使用各种形式的简写
```
Comment (core-man) [inline source/template/style.rst:22]:
```suggestion
   - 文件名应尽量使用单词全称，避免使用各种形式的简写
   - 若文件名中含多个单词，应使用连字符 (hyphen) ``-`` 连接
```

File: source/template/introduction.rst (hunk 2 above)
Comment (seisman) [reply source/template/introduction.rst:4]: Short answer: to remove whitespace between Chinese characters.

Without `\ `:

image

With `\ `:
image

File: source/template/style.rst [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+   - 索引文件 (``index.rst``): 列举软件和教程的元信息
+   - 简介 (``introduction.rst``): 主要用途、功能、特色、缺点、相关文献及引用方式
+   - 安装 (``install.rst``): 安装说明
+   - 其它可选教程
+
+     - 使用说明
+     - 实战练习
+
+3. 关于文件名的规则：
```
Comment (core-man) [inline source/template/style.rst:18]:
```suggestion
3. 文件的命名规则：
```

File: source/template/style.rst [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+   - 文件名应尽量使用单词全称，避免使用各种形式的简写
+
+文件风格
+--------
+
+1.  所有教程均采用 `reStructuredText <https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html>`__
+    语言写成
+2.  节标题、二级节标题和三级节标题，分别用 ``=``、``-`` 和 ``^`` 符号标识
+3.  所有 Bash 命令前应加上 shell 提示符 ``$`` 以表示该命令为 shell 命令
+4.  中文字符与英文字符之间应加上空格，如 ``中文 ABC 中文`` 而非 ``中文ABC中文``
```
Comment (core-man) [inline source/template/style.rst:31]:
```suggestion
4.  中文字符与英文字符和数字之间应加上空格，如 ``中文 ABC 中文`` 而非 ``中文ABC中文``，
 ``中文 123 中文`` 而非 ``中文123中文``
```

File: source/template/style.rst (hunk 1 above)
Comment (core-man) [inline source/template/style.rst:33]:
```suggestion
5.  图片应在保证清晰度的前提下尽可能小。可以使用 `squoosh 在线工具 <https://squoosh.app/>`__
    进行压缩。
```

**Comment by seisman** [comment]
@core-man It would be better if you can use "Start a review" button, so that I will be notified only once when you finish your review, rather than being notified multiple times for your reviews.

image

**Comment by core-man** [comment]
> @core-man It would be better if you can use "Start a review" button, so that I will be notified only once when you finish your review, rather than being notified multiple times for your reviews.
> 
> image

Really. Okay~

File: source/template/style.rst [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+文件风格
+--------
+
+1.  所有教程均采用 `reStructuredText <https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html>`__
+    语言写成
+2.  节标题、二级节标题和三级节标题，分别用 ``=``、``-`` 和 ``^`` 符号标识
+3.  所有 Bash 命令前应加上 shell 提示符 ``$`` 以表示该命令为 shell 命令
+4.  中文字符与英文字符和数字之间应加上空格，如 ``中文 ABC 中文`` 而非 ``中文ABC中文``，
+    ``中文 123 中文`` 而非 ``中文123中文``
```
Comment (core-man) [inline source/template/style.rst:32]:
```suggestion
4.  中文字符与英文字符和数字之间应加上空格，如 ``中文 ABC 中文`` 而非 ``中文ABC中文``，
    ``中文 123 中文`` 而非 ``中文123中文``
5.  标点符号采用全角，如 ``，``、``。``、``：``、``、``、``？`` 等。
    标点符号与中文字符、英文字符以及数字之间不需加空格。
```

File: source/template/style.rst [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+5.  图片应在保证清晰度的前提下尽可能小。可以使用 `squoosh 在线工具 <https://squoosh.app/>`__
```
Comment (core-man) [inline source/template/style.rst:33]:
```suggestion
6.  图片应在保证清晰度的前提下尽可能小。可以使用 `squoosh 在线工具 <https://squoosh.app/>`__
```

**Review by core-man — APPROVED** [review]

**Pull request merged**